Packs/Reco/Integrations/Reco/Reco.py

```diff
@@ -0,0 +1,319 @@
+            demisto.info(f"Validate API key Error: {str(e)}")
+
+        demisto.info(f"done fetching RECO incident assets, fetched {len(result)} assets.")
+        return result
+
+    def validate_api_key(self) -> str:
+        """
+        Validate API key
+        :return: bool
+        """
+        demisto.info("Validate API key, enter")
+        invalid_token_string = "Invalid token"
+        try:
+            response = self.client._http_request(
+                method="POST",
+                url_suffix="/incident-tables/tables",
+                timeout=RECO_API_TIMEOUT,
+            )
+            if response.get("listTablesResponse") is None:
+                demisto.info(f"got bad response, {response}")
+            else:
+                demisto.info(f"got good response, {response}")
+                return "ok"
+        except Exception as e:
+            demisto.info(f"Validate API key Error: {str(e)}")
+        return invalid_token_string
+
+
+def parse_table_row_to_dict(alert: List[Dict[str, Any]]) -> Dict[str, Any]:
+    if alert is None:
+        return {}
+
+    alert_as_dict = {}
+    for obj in alert:
+        key = obj.get("key", None)
+        value = obj.get("value", None)
+        if key is None:
+            continue
+        if value is None:
+            continue
+        obj[key] = base64.b64decode(value).decode("utf-8")
+        # Remove " from the beginning and end of the string
+        obj[key] = obj[key].replace('"', "")
+        if key in ["updated_at", "created_at", "event_time"]:
+            try:
+                parsed_time = datetime.strptime(obj[key], RECO_TIME_FORMAT)
+            except Exception:
+                parsed_time = datetime.strptime(obj[key], DEMISTO_OCCURRED_FORMAT)
+            if parsed_time:
+                obj[key] = parsed_time.strftime(DEMISTO_OCCURRED_FORMAT)
+        if key == "risk_level":
+            obj[key] = int(obj[key])
+        alert_as_dict[key] = obj[key]
+
+    return alert_as_dict
+
+
+def enrich_incident(reco_client: RecoClient, single_incident: Dict[str, Any]) -> Dict[str, Any]:
+    alert_as_dict = parse_table_row_to_dict(single_incident.get("cells", {}))
+    incident_id = alert_as_dict.get(RECO_INCIDENT_ID_FIELD, None)
+    if incident_id is not None:
+        alert_as_dict["assets"] = reco_client.get_incidents_assets(incident_id)
+
+    return {
+        "name": alert_as_dict.get("incident_description", ""),
+        "occurred": alert_as_dict.get("event_time", ""),
+        "dbotMirrorId": alert_as_dict.get("incident_id", ""),
+        "rawJSON": json.dumps(alert_as_dict),
+        "severity": map_reco_score_to_demisto_score(
+            reco_score=alert_as_dict.get("risk_level", DEMISTO_INFORMATIONAL)
+        ),
+    }
+
+
+def map_reco_score_to_demisto_score(
+        reco_score: int,
+) -> Union[int, float]:  # pylint: disable=E1136
+    # demisto_unknown = 0  (commented because of linter issues)
+    demisto_informational = 0.5
```

Comment:
You have const for this at the top of the file.

1. Better to import these consts from demisto if we can
2. If not, be consistent, and define all of them as consts at the top of the file